<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutSoftwareDialogClass</class>
 <widget class="QDialog" name="AboutSoftwareDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于软件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
margin-left:0px;
margin-top:0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
max-height:60px;
margin-left:0px;
margin-top:0px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="softwareIconLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
min-height:50px;
max-height:50px;
min-width:50px;
max-width:50px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../resource.qrc">:/icon/resources/icons/MaterialSymbols18UpRatingOutline.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="softwareHeaderLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
min-height:50px;
max-height:50px;
font-size:30px;
}</string>
        </property>
        <property name="text">
         <string>画皮的百宝箱</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="versionLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
	min-height:30px;
min-width:100px;
max-height:30px;
max-height:100px;
margin-left:15px;
}</string>
       </property>
       <property name="text">
        <string>版本</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="versionNumLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(100, 100, 100);
}</string>
       </property>
       <property name="text">
        <string>1.1.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="authorLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
	min-height:30px;
min-width:100px;
max-height:30px;
max-height:100px;
margin-left:15px;
}</string>
       </property>
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="UrlLabel" name="authorNameLabel">
       <property name="toolTip">
        <string>https://space.bilibili.com/19232194464</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
            color: blue;
            cursor: pointer;
font-size:13px;
        }
QLabel:hover {
text-decoration:underline;
      color:red;
}</string>
       </property>
       <property name="text">
        <string>HuaPi</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="websiteLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
	min-height:30px;
min-width:100px;
max-height:30px;
max-height:100px;
margin-left:15px;
}</string>
       </property>
       <property name="text">
        <string>官网</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="UrlLabel" name="websiteUrlLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
            color: blue;
            cursor: pointer;
font-size:13px;
        }
QLabel:hover {
text-decoration:underline;
      color:red;
}</string>
       </property>
       <property name="text">
        <string>https://huapi2d.us.kg/</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="installLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
	min-height:30px;
min-width:100px;
max-height:30px;
max-height:100px;
margin-left:15px;
}</string>
       </property>
       <property name="text">
        <string>安装路径</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="OpendirLabel" name="installPathLabel">
       <property name="styleSheet">
        <string notr="true">QLabel {
            color: blue;
            cursor: pointer;
font-size:13px;
        }
QLabel:hover {
text-decoration:underline;
      color:red;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="resourceLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:rgb(0, 0, 0);
	min-height:30px;
min-width:100px;
max-height:30px;
max-height:100px;
margin-left:15px;
}</string>
       </property>
       <property name="text">
        <string>当前资源</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="FilesTreeWidget" name="treeWidget">
       <property name="columnCount">
        <number>1</number>
       </property>
       <column>
        <property name="text">
         <string notr="true">文件资源</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>UrlLabel</class>
   <extends>QLabel</extends>
   <header>customize/customizedLabel/urllabel.h</header>
  </customwidget>
  <customwidget>
   <class>OpendirLabel</class>
   <extends>QLabel</extends>
   <header>customize/customizedLabel/opendirlabel.h</header>
  </customwidget>
  <customwidget>
   <class>FilesTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>customize/customizedFilesTree/filestreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
